source/openshift.rst

```diff
@@ -3,32 +3,32 @@ Install Percona XtraDB Cluster on OpenShift
 
 Installing Percona XtraDB Cluster on OpenShift includes two steps:
 
-* Installing the Percona Operator for Percona XtraDB Cluster,
+* Installing the Percona Distribution for MySQL Operator,
 * Install Percona XtraDB Cluster using the Operator.
 
 Install the Operator
 --------------------
 
-You can install Percona Operator for Percona XtraDB Cluster on OpenShift using the `Red Hat Marketplace <https://marketplace.redhat.com>`_ web interface or using the command line interface.
+You can install Percona Distribution for MySQL Operator on OpenShift using the `Red Hat Marketplace <https://marketplace.redhat.com>`_ web interface or using the command line interface.
 
 Install the Operator via the Red Hat Marketplace
 ************************************************
 
 1. login to the Red Hat Marketplace and register your cluster `following the official instructions <https://marketplace.redhat.com/en-us/workspace/clusters/add/register>`_.
 
-2. Go to the `Kubernetes Operator for Percona XtraDB Cluster <https://marketplace.redhat.com/en-us/products/percona-kubernetes-operator-for-percona-server-for-xtradb-cluster>`_ page and click the `Free trial` button:
+2. Go to the `Percona Distribution for MySQL Operator <https://marketplace.redhat.com/en-us/products/percona-kubernetes-operator-for-percona-server-for-xtradb-cluster>`_ page and click the `Free trial` button:
 
    .. image:: img/marketplace-operator-page.png
       :align: center
-      :alt: Percona Percona XtraDB Cluster Operator on Red Hat Marketplace
+      :alt: Percona Distribution for MySQL Operator on Red Hat Marketplace
```

Comment:
we need to rename the operator on the marketplace.
@fiowro please sync with @cap1984